src/pyggdrasil/interface/__init__.py

```diff
@@ -84,74 +82,3 @@ def append(
         self.iterations = jnp.append(self.iterations, iteration)
         self.trees.append(tree)
         self.log_probabilities = jnp.append(self.log_probabilities, log_probability)
-
-
-@dataclass
-class AugmentedMcmcData(PureMcmcData):
-    """Distance/Similarity and True Tree boolean enhanced MCMC data
-
-    intended to be human-readable and easy to use for plotting"""
```

Comment:
Abandoning this class. 

no need to store all these metrics - can just sav them to files directly.